flocker/common/test/test_script.py

```diff
@@ -300,3 +301,34 @@ def test_error(self, logger):
                    u'last):\nFailure: exceptions.ZeroDivisionError: onoes\n')
         assertHasMessage(self, logger, TWISTED_LOG_MESSAGE,
                          dict(error=True, message=message))
+
+
+class EliotStdoutTests(TestCase):
+    """
+    Tests for ``eliot_to_stdout``.
+    """
+    class StubStdout(object):
+        """
+        Stub stand-in for stdout ``write`` and ``flush`` methods.
+        Holds some bytes in memory and returns them.
+        """
+        flushed_data = b''
+        buffered_data = b''
+
+        def write(self, message):
+            self.buffered_data = self.buffered_data + message
+
+        def flush(self):
+            self.flushed_data = self.buffered_data
+            self.buffered_data = b''
+
+    def test_destination_stdout(self):
+        """
+        Eliot messages are written to stdout.
+        """
+        fake_stdout = self.StubStdout()
+        message_formats = {'flocker:eliot:test': 'Running %(command)s\n'}
+        eliot_to_stdout(message_formats, {}, stdout=fake_stdout)
```

Comment:
Does this need to be reversed in some way so that the rest of the testsuite
doesn't print eliot logs to stdout?